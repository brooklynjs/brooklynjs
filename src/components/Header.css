header {
  background: #222;
  border-top: 4px solid #FFF;
  font-size: 72px;
  font-weight: bold;
  line-height: 68px;
  margin: 20px auto;
  position: relative;
  text-align: left;
  width: 500px;
}

header span {
  background: #F7DF1E;
  border-radius: 50%;
  color: #222;
  font-size: 60px;
  height: 72px;
  line-height: 72px;
  position: absolute;
  text-align: center;
  top: 7px;
  width: 72px;
}

header .j {
  right: 80px;
}

header .s {
  right: 0;
}
