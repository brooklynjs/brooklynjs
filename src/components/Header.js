import React from 'react';
import './Header.css';

const Header = () => {
  return (
    <header>
      Brooklyn
      <span className="j">J</span>
      <span className="s">S</span>
    </header>
  );
};

export default Header;
