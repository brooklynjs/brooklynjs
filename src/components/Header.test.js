import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Header from './Header';

it('renders', () => {
  const tree = ReactTestRenderer.create(<Header />).toJSON();

  expect(tree).toMatchSnapshot();
});
