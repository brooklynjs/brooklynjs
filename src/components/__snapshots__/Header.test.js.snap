// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<header>
  Brooklyn
  <span
    className="j"
  >
    J
  </span>
  <span
    className="s"
  >
    S
  </span>
</header>
`;
